framework/encode/handle_unwrap_memory.h

```diff
@@ -71,6 +72,7 @@ class HandleUnwrapMemory
             next_buffer = &buffers_[next_index];
             next_buffer->clear();
             next_buffer->insert(next_buffer->end(), data, data + len);
+            std::copy(data, data + len, std::back_inserter(*next_buffer));
```

Comment:
I think you may have wanted to delete the previous line with the `insert`.

Reply:
fixed